psfws/gen_params.py

```diff
@@ -0,0 +1,371 @@
+"""Class to generate realistic input parameters for atmospheric PSF sims."""
+
+import numpy as np
+import pickle
+import pandas as pd
+import pathlib
+from psfws import utils
+
+
+class ParameterGenerator():
+    """Class to generate realistic input parameters for atmospheric PSF sims.
+
+    This class uses two inputs: NOAA Global Forecasting System (GFS) outputs
+    and local wind measurements at the observatory. The repo already contains
+    these data for Cerro Pachon, and all defaults are set up to match this
+    location. Use of the code to generate parameters for Cerro Pachon (and
+    Cerro Telolo, nearby) is straightforward, but for use at other
+    observatories, users must supply input data: instructions
+    for downloading NOAA datasets and formatting telemetry are in the
+    repository README.
+
+    Attributes
+    ----------
+    gfs_winds : pandas dataframe
+        GFS wind values (matched w telemetry), with DateTime as index and
+        columns 'u', 'v', 'speed', 'dir'. Each entry in the DataFrame is a
+        ndarray of values for each altitude, with speed/velocity components
+        in m/s and directions in degrees. The u/v components of velocity
+        correspond to north/south winds, respectively, and the wind direction
+        is given as degrees west of north.
+    telemetry : dict of ndarrays
```

Comment:
These have since been replaced with a pandas dataframe, but this comment makes me wonder if it would be easier, from the user end, if the output parameter dict from get_parameters() was replaced by a named tuple.